2226/2/raper.rb

```diff
@@ -0,0 +1,38 @@
+# This class is used for single raper methods
+class Raper
+  attr_reader :raper_name
+
+  def initialize(raper_name)
+    @raper_name = raper_name
+  end
+
+  def number_of_battles
+    Battle.new(@raper_name).number_of_battles
```

Comment:
Use delegate pattern for things like this. But Actually what u are doing is totally wrong. You can remove rapper class from this program and use Battle class only.